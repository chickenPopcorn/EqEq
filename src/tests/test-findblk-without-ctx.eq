myVal = 42; 
globalVal = 32; /* in the "global" context */

/* `find` doesn't prefix with a "context" */
find myVal {
  print("myVal is defined on the global context as %.0f\n", myVal);
  print("globalVal is defined on the global context as %.0f\n", globalVal);
}

find globalVal {
  print("globalVal is defined on the global context as %.0f\n", globalVal);
}
