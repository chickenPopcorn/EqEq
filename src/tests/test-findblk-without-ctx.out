myVal is defined on the global context as 42
globalVal is defined on the global context as 32
globalVal is defined on the global context as 32
